js/modules/loki_file_server_api.d.ts

```diff
@@ -0,0 +1,14 @@
+interface DeviceMappingAnnotation {
+  isPrimary: string;
+  authorisations: Array<{
+    primaryDevicePubKey: string;
+    secondaryDevicePubKey: string;
+    requestSignature: string; // base64
+    grantSignature: string; // base64
```

Comment:
what about a custom type for that so we know this is a base64 whenever this is used?

Reply:
Make sense